main.cpp

```diff
@@ -507,6 +587,15 @@ void OscillatorSection::Waveform::fillEntireWaveTable(char transformationTypeInd
     std::cout << "Wavetable filled with transformation algorithm No: " << transformationTypeIndex << std::endl;
 }
 
+void OscillatorSection::Waveform::shiftPhase(int phase)
+{
+    while (initialPhase - phase > 0)
+        {
+            initialPhase -= phase;
+            std::cout << "Phase shifted for " << phase << " degrees\n";
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise